addons/project/views/project_project_views.xml

```diff
@@ -105,8 +105,7 @@
                                     </div>
                                     <!-- the alias contact must appear when the user start typing and it must disappear
                                         when the string is deleted. -->
-                                    <field name="alias_contact" class="oe_inline" string="Accept Emails From"
-                                           attrs="{'invisible': ['|', ('alias_name', '=', ''), ('alias_name', '=', False)]}"/>
+                                    <field name="alias_contact" class="oe_inline" string="Accept Emails From" attrs="{'invisible': [('alias_name', '=', False)]}"/>
```

Comment:
Can you explain why it is needed? 🤔

Reply:
In attributes both conditions were similar and they had the same impact so removed one of them

Reply:
In fact, not really, `('alias_name', '=', False)` checks is the field is `NULL` in the DB, that is, if the field is not yet set in the record we see. And `('alias_name', '=', '')` checks if `alias_name == ''` in the db, for instance, if the user erases the content of this. field, the value recorded in the db will be an empty string instead of NULL value. 🙂

Reply:
Got it Thanks :+1: